docs/cvm.md

```diff
@@ -1,7 +1,250 @@
-# Commercial Vehicle Model
+#### Introduction
+The SANDAG Commercial Vehicle Model (CVM) simulates the weekday demand patterns of commercial 
+vehicle movements throughout the San Diego region. The CVM is an important part of the complete travel 
+demand modeling system for the region, representing a market of travel that dominates the middle part of 
+most weekdays and has been steadily growing as consumer demands for home deliveries and personal 
+services have increased.
+
+The primary sources of data for developing the CVM were the 2022 SANDAG Commercial Vehicle
+Establishment and TNC-driver surveys, which focused on goods, services, and maintenance trips, and 
+obtained travel diaries from employees of these establishments whose jobs involve routine travel for either 
+goods pickup and delivery or for service provision. The TNC driver survey used an identical travel diary 
+format to the Establishment survey, the difference being that individual TNC drivers were surveyed as their 
+own establishments who worked on behalf of an online pickup and delivery service. The Establishment and 
+TNC surveys provided detailed travel pattern data for individual drivers and vehicles, which formed the basis 
+for estimating and calibrating model components. The categorical definitions of attribute variables in the two 
+surveys set the possibilities for segmentation of the model system, such as establishment industry sectors;
+trip origin and destination purposes, land uses, and place types; and vehicle types.
+
+The geographic scope of the CVM are internal-to-internal trip movements. The market scope of the model 
+includes commercial goods movements (pickup and deliveries) as well as trips made for commercial and 
+public services. Trips made for other purposes, namely maintenance and personal, are also included in the 
+CVM if these trips are made in the context of a commercial vehicle tour pattern. The CVM explicitly 
+distinguishes between residential and non-residential customer types, and between three vehicle types—
+light, medium, and heavy—consistent with the definitions used in the Establishment Survey.
+
+The CVM does not cover the types of work-related travel that would be expected to be covered in the ABM3
+Resident model, namely workers traveling for meetings, sales calls, out-of-town travel, and similar activities.
+The CVM also does not model long-distance freight truck movements that enter and exit the region, which 
+are covered by the Heavy Truck Model (HTM).
+
+The HTM covers long-distance freight movements into and out of San Diego County. The source of the 
```

Comment:
The HTM part could be removed from here.

Reply:
Removed HTM